src/index.html

```diff
@@ -7,8 +7,78 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+
+      <div class="container-logo">
+        <a href="#home" class="logo">
+          <img src="images/logo.png" alt="MOYO">
+        </a>
+      </div>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#Apple" class="nav__link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Samsung" class="nav__link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Phones" class="nav__link">
+              Смартфоны и телефоны
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#PC"
+            data-qa="hover"
+            class="nav__link"
+            >
```

Comment:
Fix your indentation here. If you have more than 3 attributes, then each of them should be on a newline
```suggestion
            <a 
              href="#PC"
              data-qa="hover"
              class="nav__link"
            >
```